### PR TITLE
Do not run triton tests on sm86

### PR DESCRIPTION
**Pull request by malfet**
As its broken right now and nobody care to fix it, see this test run for example: https://hud.pytorch.org/pytorch/pytorch/commit/d36c284d1446cb250178f8e89fff9b342ee1a5a9

### CODE REVIEW

**Review by soumith — APPROVED** [review]

**Review by albanD — APPROVED** [review]

File: .jenkins/pytorch/test.sh [hunk 1]
```diff
@@ -699,12 +699,16 @@ elif [[ "${TEST_CONFIG}" == *dynamo* && "${SHARD_NUMBER}" == 2 && $NUM_TEST_SHAR
 elif [[ "${SHARD_NUMBER}" == 1 && $NUM_TEST_SHARDS -gt 1 ]]; then
   test_without_numpy
   install_torchvision
-  install_triton
+  if ! [[ "${BUILD_ENVIRONMENT}" == *sm86 ]]; then
+  vvinstall_triton
```
Comment (jansel) [inline .jenkins/pytorch/test.sh:703]: typo

**Review by jansel — CHANGES_REQUESTED** [review]
Isn't it just two tests that are failing?  Perhaps we should just skip those two tests on sm86, rather than all 1000+ tests.

**Comment by malfet** [comment]
> Isn't it just two tests that are failing? Perhaps we should just skip those two tests on sm86, rather than all 1000+ tests.

No, it's not just two test(i've tried to disable few already), dynamo is crashing with some sort of exception(see https://github.com/pytorch/pytorch/actions/runs/3270773917/jobs/5380027690 ) that I have no bandwith to debug and want to return CI back to sanity. 
IMO there are 2 path for doing that:
- Disabling testing on sm86 and let TorchDynamo team figure out what is going on
- Disabling sm86 shard

**Pull request closed without merging**